changelog.txt

```diff
@@ -5,6 +5,7 @@
 * Add - Payment Request Button support for US merchants (Apple Pay, Google Pay, Microsoft Pay, and the browser standard Payment Request API).
 * Update - Not passing level3 data for non-US merchants.
 * Add - REST endpoint for fetching account data.
+* Add - Deposits pagination and sorting
```

Comment:
```suggestion
* Add - Deposits list pagination and sorting.
```

Reply:
Done in f93307f22ca0946f84534150ff64e50e391bbd1e